src/features/events/components/EventParticipantsList.tsx

```diff
@@ -42,9 +43,26 @@ const EventParticipantsList: FC<EventParticipantsListProps> = ({
         description={messages.eventParticipantsList.descriptionBooked()}
         model={model}
         orgId={orgId}
-        rows={model.getParticipants().data ?? []}
+        rows={model.getBookedParticipants()}
         title={messages.eventParticipantsList.bookedParticipants()}
+        type="booked"
       />
+      {model.getNumCancelledParticipants() > 0 && (
+        <ParticipantListSection
+          chipColor={model.getParticipantStatus()}
+          chipNumber={
+            model.getNumAvailParticipants() +
+            '/' +
+            data.num_participants_required
+          }
```

Comment:
This should be the number of cancelled participants, not identical to the participants section.

Reply:
Makes sense, missed that. Fixed.